src/dataDisplay/table/table.d.ts

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+export interface TablePropTypes {
+    /**
+     * Basic (default) styling.
+     */
+    basic?: boolean;
+    /**
+     * If `true`, all columns will be divided.
+     */
+    celled?: boolean;
+    /**
+     * The content of the Table.
+     */
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the Table.
+     */
+    classes: {
+        tableSticky: string;
+        tableStickyColumns: string;
+    };
+    /**
+     * Add additional classes to the Table.
+     */
+    className: string;
+    /**
+     * If `true`, applies `width: auto` to the Table.
+     */
+    collapsing: boolean;
+    /**
+     * If `true`, the far left column is divided from the rest of the columns.
+     */
+    definition: boolean;
+    /**
+     * If `true`, columns will be evenly spaced.
+     */
+    fixed: boolean;
+    /**
+     * Change the font size of text in the Table.
+     */
+    fontSize: 'xxlarge' | 'xlarge' | 'large' | 'medium' | 'small' | 'xsmall' | 'xxsmall';
+    /**
+     * Deprecated prop.
+     */
+    fullWidth: boolean;
+    /**
+     * Add an id to the Table.
+     */
+    id: string;
+    /**
+     * Table rows and cells can be highlighted as if they have been selected.
+     */
+    selectable: boolean;
+    /**
+     * Truncate copy within cells in the Table
+     */
+    singleLine: boolean;
+    /**
+     * Change the cells vertical size in the Table.
+     */
+    size: 'l' | 'large' | 'm' | 'medium' | 's' | 'small';
+    /**
+     * Deprecated prop.
+     */
+    stackable: boolean;
+    /**
+     * A table can have one or more columns defined as 'sticky' so that they stay fixed while the
+     * user horizontally scrolls to see the remaining columns.
+     */
+    stickyColumnCount: number;
```

Comment:
That would be a breaking change, and we'd have to rename the prop in the component as well (this is just the corresponding TypeScript declaration).

Fortunately, it doesn't seem like it would be too hard to update HC client accordingly:

![image](https://user-images.githubusercontent.com/4349658/148254510-c3700c23-4b20-4089-ac93-4f565b389f21.png)

@cbrewer1689 Your call.  IMO this is not blocking and discretionary.

Reply:
I am going to leave this prop alone for now. Maybe down the road would be good.